### ISSUE TITLE
Email: IBM Comments on WCAG 3.0 Draft (5 - Usability and understandability of the standard itself)

### ISSUE THREAD

**Issue by cwadamsoforacle** [w3c/silver#499]
[Comment from email:](https://lists.w3.org/Archives/Public/public-agwg-comments/2021Mar/0005.html)
Usability and understandability of the standard itself
•	Diagrams of the structure and organization of the document could use some updates for usability:
o	Section 2 Structure of these guidelines Figure 1 doesn't have all of the parts of the methods as shown in Figure 2. Missing "Introduction" and "Resources". Is this intentional?
o	Section 2.3 Methods structure is missing a couple of sub-sections based on Figure 2. There should be a subsection on the Introduction tab and what you can find there as well as a subsection on the Resources tab.
o	Section 3 Additional Documentation and Scoring: The changes in terminology between WCAG 2.x and WCAG 3 are sometimes confusing.  Should there be a readily linkable section or inline definition that shows equivalent language, similarities in language, or a new definition/use of a WCAG 2.x term.  Or, avoid unnecessarily creating new terms. 
•	Figure 3 is confusing for the following reasons: 
o	Why do some of the boxes in the figure have heavy lines and others don't. Those aren't the documents, so what is the visual treatment trying to convey?
o	From this diagram it seems there are three kinds of scoring: test scoring, outcome scoring and functional category scoring. Is that correct, or are we misreading the diagram? 
o	Combining document structure/relationships with the scoring relationship is confusing, especially since there's no indication in the diagram of what items represent documents vs. what items are needed for scoring. Separating scoring from documentation relationships into two diagrams may help - with a full text explanation of each.
o	The diagram is also confusing because How-to's have an "Outcomes" tab, which lists the outcomes, but has no connection in the diagram to outcomes.
o	If guidelines are normative, Functional Needs and functional categories seem out of place, as they are supporting information providing supportive context of the requirement. Perhaps they should be moved to the non-normative part of the documentation.
•	Question from status section: Are there usability improvements that would make WCAG 3 easier to use and find information? Is the structuring of the How-To, outcomes, and methods clear? Are there improvements to structure and style that you would like to see? Is there another way we can make content more usable? 

Response: There are a lot of resources, which does confuse things a bit. There are Functional Needs, Guidelines, Outcomes, Methods, and How-to's. The navigation between the documents is a little confusing and having this number of documents may make maintenance of WCAG 3 more time consuming.  It is difficult to communicate to those unfamiliar with them which document(s) are best to fully understand a requirement and how to implement it. Wondering if it might be better to combine methods and how-to’s into one resource and potentially guidelines and outcomes into one resource. Then there’s less chance for getting lost or confused about which resource you’re in or how to get to the related resource.
•	Section 4. Normative requirements should be earlier in the document since this term is used in sections above with no pointers/links to this section where it describes what the term means. Also suggest that the normative items be provided in a list format rather than prose. Lists are quicker and easier to parse.

**Comment by cwadamsoforacle** [comment]
Thank you for your comment. Project members are working on your comment. You may see discussion in the comment thread and we may ask for additional information as we work on it. We will mark the official response when we are finished and close the issue.

**Comment by cwadamsoforacle** [comment]
[IBM comments on the WCAG 3 FPWD -March 2.docx](https://github.com/w3c/silver/files/6174024/IBM.comments.on.the.WCAG.3.FPWD.-March.2.docx)
Forgot to add attachment to original post.

**Comment by cwadamsoforacle** [comment]
Proposed response:

This content has been updated with the release of the latest draft of WCAG 3, which can be found here:  [W3C Accessibility Guidelines (WCAG) 3.0](https://www.w3.org/TR/wcag-3.0/).  The Accessibility Guidelines Working Group attempted to address these concerns in this latest draft.  Should you feel that these issues remain in the current draft, please log a new issue here:  [New Issue · w3c/wcag3](https://github.com/w3c/wcag3/issues/new).
